cmd/config.go

```diff
@@ -22,25 +22,31 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/pilosa/pilosa/ctl"
+	"github.com/pilosa/pilosa/server"
 )
 
 var Conf *ctl.ConfigCommand
 
 func NewConfigCommand(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
 	Conf = ctl.NewConfigCommand(os.Stdin, os.Stdout, os.Stderr)
+	Server := server.NewCommand(stdin, stdout, stderr)
```

Comment:
Yes, that line starts with the default configuration but it gets passed to `BuildServerFlags()` which attaches to `cobra` and updates the default configuration based on a `--config` file passed in, command line flags, etc.

For example, I wrote out a `/tmp/pilosa.conf`:

```
data-dir = "~/.pilosa2"
bind = "localhost:10102"
max-writes-per-request = 5002

[cluster]
  poll-interval = "3m0s"
  replicas = 2
  hosts = [
    "localhost:10102",
    "localhost:10103",
  ]

[anti-entropy]
  interval = "20m0s"

[profile]
  cpu = "2"
  cpu-time = "40s"

[plugins]
  path = "test"
```

And then passed it to the `config` command it (while overriding the `--data-dir`):

```sh
$ pilosa --config /tmp/pilosa.conf config --data-dir "/foo/bar"
data-dir = "/foo/bar"
host = "localhost:10102"
log-path = ""
max-writes-per-request = 5002

[anti-entropy]
  interval = 1200000000000

[cluster]
  gossip-seed = ""
  hosts = ["localhost:10102","localhost:10103"]
  internal-hosts = []
  internal-port = ""
  polling-interval = 180000000000
  replicas = 2
  type = "static"

[plugins]
  path = "test"
```